WordPressKit/CommentServiceRemoteREST.m

```diff
@@ -398,13 +398,23 @@ - (void)getLikesForCommentID:(NSNumber *)commentID
     NSString *path = [NSString stringWithFormat:@"sites/%@/comments/%@/likes", self.siteID, commentID];
     NSString *requestUrl = [self pathForEndpoint:path
                                      withVersion:ServiceRemoteWordPressComRESTApiVersion_1_2];
+    NSDictionary *params = @{ @"meta": @"site,comment"};
 
     [self.wordPressComRestApi GET:requestUrl
-                       parameters:nil
+                       parameters:params
                           success:^(id responseObject, NSHTTPURLResponse *httpResponse) {
         if (success) {
-            NSArray *jsonUsers = responseObject[@"likes"] ?: @[];
-            success([self remoteUsersFromJSONArray:jsonUsers]);
+            NSNumber *commentID = responseObject[@"meta"][@"data"][@"comment"][@"ID"];
+            NSNumber *siteID = responseObject[@"meta"][@"data"][@"site"][@"ID"];
```

Comment:
If you like, `numberForKeyPath` is alternate syntax vs walking the arrays.
```
[responseObject numberForKeyPath:@"meta.data.comment.ID"]
[responseObject numberForKeyPath:@"meta.data.site.ID"]
```
A question tho:  Line 398 shows we already have the siteID and commentID. 
```
NSString *path = [NSString stringWithFormat:@"sites/%@/comments/%@/likes", self.siteID, commentID];
```
Is it necessary to retrieve them from the response?

Reply:
Only out of paranoia to be sure the IDs matched the response. I'll remove it.